IHP/IDE/SchemaDesigner/Parser.hs

```diff
@@ -546,6 +546,7 @@ alterType = do
 
 -- | ALTER TABLE users ALTER COLUMN email DROP NOT NULL;
 --  ALTER TABLE users ALTER COLUMN email SET NOT NULL;
+--  ALTER TABLE users ALTER COLUMN email SET DEFAULT 'value';
```

Comment:
Could you add a test case for this query in the Parser and Compiler tests? While the new branches are called from the migration tests, it would still be useful to have a test in the parser and compiler as well 👍